OpenUpSA/wazimap-ng
# Fix point data upload when there is space in csv headers

## Description
strip the white space for csv headers for point uploads

## Related Issue
#218

## How to test it locally
Add space in header in starting or ending for any header of a point csv and try to upload it.
Upload should fail.

## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Commits


### Code Quality


### Testing
